src/app/public/modules/vertical-tabset/fixtures/vertical-tabset-no-group.component.fixture.html

```diff
@@ -1,5 +1,7 @@
 <div class="vertical-tabset-test-indexchange">current index = {{ currentIndex }}</div>
-<sky-vertical-tabset (activeChange)="indexChanged($event)">
+<sky-vertical-tabset
+  [maintainTabContent]="maintainTabContent"
+  (activeChange)="indexChanged($event)">
```

Comment:
Closing bracket needs to be on a new line.

Reply:
Done